tutorials/gcs-cf-auth/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Google Cloud Functions Access Control
```

Comment:
We use sentence case for titles and headings, so no need to capitalize 'access control' here.